schema/brew-fast/magicvariants.json

```diff
@@ -69,11 +69,16 @@
 								},
 								"property": {
 									"type": "string"
+								},
+								"customProperties": {
+									"description": "An unrestricted area for custom properties to be stored.",
+									"type": "object",
+									"items": {
+										"type": "boolean"
+									}
```

Comment:
So when we have `customProperties` in the `requires` / `excludes` area. I expect those are restricted to boolean values like currently.
This is my expectation for what this will look like, just want to check that's lining up with how you see the site being set up?
![image](https://github.com/TheGiddyLimit/5etools-utils/assets/52298102/3443c1dd-1a37-4122-9c3e-787b9ac820e3)

Reply:
makes sense to me! Have you tested to check it works, or is there some tuning-up required?

Reply:
I've had a go, it doesn't appear to work unfortunately. 
Here's a basic test file.
https://gist.githubusercontent.com/revilowaldow/98fe44b73eb83bd89ff32fa8462bb05a/raw/9f606d5f324a2f7c3a90935d14e1806ffd15001e/Various;%2520Lyra's%2520Legendarium.json

Reply:
should be working in next release